src/app/auth/authentication.service.ts

```diff
@@ -138,6 +146,7 @@ export class AuthenticationService {
       let options = new RequestOptions({ headers: headers });
       return this.http.get(tokenUrl, options)
         .map(response => processToken(response))
+        .do(token => localStorage.setItem(broker + '_token', token.access_token))
```

Comment:
@pmuir - the processToken line above never actually called the function that was storing them (and would have overridden the keycloak stored token even if it did).  I did see the tokens successfully saved in storage with this intermediate processing step however.